resources/GitDsc/GitDsc.psd1

```diff
@@ -8,130 +8,129 @@
 
 @{
 
-# Script module or binary module file associated with this manifest.
-RootModule = 'GitDsc.psm1'
+    # Script module or binary module file associated with this manifest.
+    RootModule           = 'GitDsc.psm1'
 
-# Version number of this module.
-ModuleVersion = '0.1.0'
+    # Version number of this module.
+    ModuleVersion        = '0.1.1'
```

Comment:
I just caught this but don't increment the version on any of these manifest files. We have an internal publishing pipeline that will do that for us.

Reply:
Sounds good!